insights/parsers/nfs_conf.py

```diff
@@ -0,0 +1,69 @@
+"""
+General configuration file for NFS daemons and tools - ``/etc/nfs.conf``
+========================================================================
+
+This file contains site-specific configuration for various NFS daemons and
+other processes. In particular, this encourages consistent configuration
+across different processes.
+
+NFSCONF - file ``/etc/nfs.conf``
+--------------------------------
+
+"""
+
+from insights.core.plugins import parser
+from insights.core import IniConfigFile
+from insights.specs import Specs
+
+
+@parser(Specs.nfs_conf)
+class NFSCONF(IniConfigFile):
```

Comment:
@JoySnow the name should be `NFSConf` with mixed case shouldn't it?

Reply:
Renamed to "NFSConf".